src/client/src/containers/Licenses/index.tsx

```diff
@@ -38,7 +40,7 @@ const Licenses = ({
 }: Props) => {
   const LinkRenderer = (props: any) => {
     return (
-      <a href={props.href} className={styles.link}>
+      <a href={props.href} className={styles.link} onKeyUp={keyUpHandler}>
```

Comment:
For the Licenses file, we are going to add a `Privacy Modal`. So, these anchor tags would be converted to buttons that would open up the modal on click. Can you revert the changes for this file?